### PR TITLE
fix: Programmatic navigation ParametersViewController 

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**

### CODE REVIEW

File: kDrive/UI/Controller/Menu/GenericGroupedTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import kDriveResources
+import UIKit
+
+class GenericGroupedTableViewController: UITableViewController {
```
Comment (adrien-coye) [inline kDrive/UI/Controller/Menu/GenericGroupedTableViewController.swift:22]: Maybe a `BaseGroupedTableViewController` as the notion of generics is a bit different ?

**Review by adrien-coye — COMMENTED** [review]
I did test this PR on iPhone8, all good. I'd approve this PR but I have a tiny remark.

File: kDrive/UI/Controller/Menu/GenericGroupedTableViewController.swift (hunk 1 above)
Comment (PhilippeWeidmann) [reply kDrive/UI/Controller/Menu/GenericGroupedTableViewController.swift:22]: I agree 👍

**Review by adrien-coye — APPROVED** [review]
Thanks for the changes.

**Pull request merged**